### PR TITLE
Transcripts - Update styles

### PR DESCRIPTION
**Pull request by ashiagr**
## Description
This updates 
- Error layout UI & styles (Figma: tzuqhOmpPVu5TjKK1XMhf5-fi-380_10286)
- Colors for loading view 
- Content padding for the transcript view (more horizontal padding for tablets)
- Search occurrences text background color (Figma: tzuqhOmpPVu5TjKK1XMhf5-fi-380_9904)

** I'll update the font later, requested it here (tzuqhOmpPVu5TjKK1XMhf5-fi-380_9904#914059552)
EDIT: See https://github.com/Automattic/pocket-casts-android/pull/2654#issuecomment-2293239636

#### Test.1 Error States UI
1. Comment out line 77 -84 of `TranscriptViewModel` and throw each exception caught below
2. Install the app
3. Play an episode with transcript (preferably choose an episode with tint e.g. https://pca.st/podcast/d560d3e0-6034-0138-97eb-0acc26574db2)
4. Open transcript view from full-screen player
5. ✅ Notice that the UI and style matches with Figma
6. Revert to the original code and reinstall the app

#### Test.2 Try again 
1. Tap on an episode with transcript for which you haven't opened the transcript view
2. Disable internet
3. Open fullscreen player and tap the transcript icon
4. ✅ Notice that No network connection error is shown
7. Enable internet
8. Tap Try again
9. ✅ Notice that transcript text is loaded 

#### Test.3 Style checks 

Check styles for 
- Loading views (when content loads and for pull-to refresh)
- Content horizontal padding (particularly on a tablet)
- Search text background

## Screenshots or Screencast 

Error

Loading

Search 

Tablet

## Checklist
- [x] ~~If this is a user-facing change, I have added an entry in CHANGELOG.md~~
- [x] Ensure the linter passes (`./gradlew spotlessApply` to automatically apply formatting/linting)
- [x] ~~I have considered whether it makes sense to add tests for my changes~~
- [x] All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- [x] Any jetpack compose components I added or changed are covered by compose previews
- [x] ~~I have updated (or requested that someone edit) [the spreadsheet](https://docs.google.com/spreadsheets/d/107jqrutZhU0fVZJ19SBqxxVKbV2NWSdQC9MFYdLiAxc/edit?usp=sharing) to reflect any new or changed analytics.~~ Analytics will be added later
 
#### I have tested any UI changes...

- [x] with different themes
- [x] with a landscape orientation
- [x] with the device set to have a large display and font size
- [ ] for accessibility with TalkBack

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please add tests for class <code>TranscriptColors</code> (or add <code>unit-tests-exemption</code> label to ignore this).</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by ashiagr** [comment]
Based on this discussion(tzuqhOmpPVu5TjKK1XMhf5-fi-380_9904#914059552), I updated the font in 8ad4de9.

Revised screenshots for content, search, and error:

Content | Search | Error
---|---|---
Screenshot_20240816_153608 | search|error

File: modules/services/localization/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1894,8 +1895,9 @@ up    <string name="player_sleep_in_one_chapter">Sleeping in 1 chapter</string>
     <string name="bookmarks_search_results_not_found">We couldn\'t find any bookmark for that search.</string>
 
     <!-- Transcripts -->
-    <string name="error_transcript_failed_to_load">Transcript failed to load</string>
-    <string name="error_transcript_format_not_supported">Transcript format not supported: %s</string>
+    <string name="error_transcript_failed_to_load">Sorry, but something went wrong while loading this transcript.</string>
+    <string name="error_transcript_format_not_supported">Sorry, but but this transcript format is not supported: %s</string>
```
Comment (mebarbosa) [inline modules/services/localization/src/main/res/values/strings.xml:1899]:
```suggestion
    <string name="error_transcript_format_not_supported">Sorry, but this transcript format is not supported: %s</string>
```

**Review by mebarbosa — APPROVED** [review]
Great job! I tested with different device sizes and themes and all worked as expected! 

I left just one small comment, but it's not a blocker

**Pull request merged**